qa/workunits/cephtool/test.sh

```diff
@@ -1601,6 +1601,12 @@ function test_mon_osd()
   expect_false ceph osd tree up down
   expect_false ceph osd tree in out
   expect_false ceph osd tree up foo
+  # make sure we reached luminous to enable crush class feature
+  ceph osd set-require-min-compat-client luminous
+  ceph osd crush class create hdd
+  ceph osd crush set-device-class osd.0 hdd
```

Comment:
@liewegas Add the test cases here but do not test this together with https://github.com/ceph/ceph/pull/16014, in which 
I changed the set-device-class cmd slightly to support multiple osd-ids.
I'll update these correspondingly once https://github.com/ceph/ceph/pull/16014 gets merged.